metadata_ingest/form_parsers.py

```diff
@@ -80,12 +79,12 @@ def parse_contactPoints(self, contactPoint):
                     'hasEmail': poc[1],
                     'hasRole': k
                 }
-                contactPoint_array.append(poc_dict)
-        return contactPoint_array
+                contact_point_array.append(poc_dict)
+        return contact_point_array
 
-    def parse_identifiersExtended(self, identifiersExtended):
-        if not identifiersExtended:
-            return identifiersExtended
+    def parse_identifiersExtended(self, identifiers_extended):
+        if not identifiers_extended:
+            return identifiers_extended
         identifiersExtended = [dict(zip(['type', 'uid'], entry.split(':')))
```

Comment:
If `identifiersExtended` is being renamed to `identifiers_extended`, make sure to rename all `identifiersExtended` variables.